model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -251,34 +251,45 @@ case class WorkspaceStatus(workspaceName: WorkspaceName, statuses: Map[String, S
 
 case class BucketUsageResponse(usageInBytes: BigInt)
 
-case class ErrorReport(source: String, message: String, statusCode: Option[StatusCode], causes: Seq[ErrorReport], stackTrace: Seq[StackTraceElement], exceptionClass: Option[Class[_]])
+case class CErrorReport(source: String, message: String, statusCode: Option[StatusCode], causes: Seq[CErrorReport], stackTrace: Seq[StackTraceElement], exceptionClass: Option[Class[_]])
```

Comment:
Also calling this `ErrorReport` made the compiler very unhappy, so I renamed it.